### ISSUE TITLE
Problems with PHP 7

### ISSUE THREAD

**Issue by gsilvestri** [lparchaeology/ARK1#1]
After upgrading to PHP 7 to test ARK v2.0 my ARK 1.2 installation stopped working: attached screenshot of the error.

![screenshot from 2017-12-10 23-40-58](https://user-images.githubusercontent.com/8915099/33810319-97a17ea0-de03-11e7-85fc-d534eac4d4a2.png)

**Comment by jlayt** [comment]
Yes, ARK1 only supports PHP5, there's far too many obsolete dependencies in it to be upgraded. In particular, the PEAR LiveUser library is very broken. You can run PHP5 and PHP7 in parallel on the same machine using PHP-FPM, see http://ark.lparchaeology.com/wiki/index.php/ARK2/Install for some hints.
